ex36 - without is_game_continue.py

```diff
@@ -0,0 +1,206 @@
+			print("================================================\n\n")
+	
+# The Crow zone
+def crow():
+	print("After leaving the chest, you continue to the path ahead.")
+	print("You see a strange short tree, with a white crow on the longest branch.")
+	print("The crow is holding a shiny key in its mouth.")
+
+	# Number of Failed attempts (max: 3)
+	crow_false_attempt = 0
+
+	while True:
+		# Remind player of choices
+		show_crow_menu()
+
+		crow_choice = input("> ")
+	
+		if crow_choice == "1":
+			print("\n================================================")
+			print("You look at the crow. The crow looks back at you. In silence.")
+			print("================================================\n\n")
+			print("\nYou feel painful in your intestines, as if something wants to get out!")
+			print("You shouldn't have stared at the crow from the beginning.\n")
+			crow_false_attempt += 1
+			count_mistake(crow_false_attempt)
+		elif crow_choice == "2":
+			print("The crow flies away, with the shiny key still in his mouth.")
+			print("Damn it! The key must have been useful later!")
+			print("Now you lost it forever!")
+			return False
+		elif crow_choice == "3":
+			print("The crow rushes into your hand to grab the purple diamond.")
+			print("He flies away to the sky, left the shiny key fall to the grass.")
+			print("You pick up the key.\n\n")
+			return True
+
+# The Great Wall
+def great_wall(chest_key, crow_key):
+	print("You stand before a very high and sturdy wall.")
+	print("The only way through the Great Wall is a locked door.")
+	print("You look into your pockets.")
+	## TODO list
+	# Write a key_selection procedure: what key you have, try out the key.
+	# Write a 100 HP wall, which you can kick until it opens.
+	# Plus some surprises if too many same actions are repeated (Hidden action).
+	if chest_key:
+		print("You have the key from the shiny chest.")
+	if crow_key:
+		print("You have the key from the white crow.")
+	
+	# A surprise later - hidden choice
+	wall_attempt = 0
+
+	while True:
+		# Remind player of choices
+		show_great_wall_menu()
+		great_wall_choice = input("> ")
+
+		if great_wall_choice == "1":
+			if chest_key:
+				print("You tried the Chest key, but the door won't open.")
+				print("Try something else.")
+			else:
+				print("You don't have the Chest Key to use. So sad.")
+				print("Try something else.")
+		elif great_wall_choice == "2:"
+			if crow_key:
+				print("You tried the Crow key, and the door is opened.")
+				print("A new journey awaits you ahead.")
+				print("TO BE CONTINUED...")
+				exit(0)
+			else:
+				print("You don't have the Crow Key! Remember something at the tree?")
+				print("Try something else.")
+		elif great_wall_choice == "3":
+			print("You cry like a baby, your guts hate you.")
+			print("But no one can help you now. Better be back.")
+
+
+		# Player rage mode. Destroy things!	(wall_attempt == 3)		
+		wall_attempt +=1
+		if wall_attempt == 3:

```

Comment:
3 is a magic number
Create a const like:
```python
MAX_WALL_ATTEMPT = 3
if wall_attempt == MAX_WALL_ATTEMPT:
```

Reply:
Got it.